packages/api/orm/entity/Clin.ts

```diff
@@ -0,0 +1,51 @@
+import { BaseEntity } from "./BaseEntity";
+import { Column, Entity, ManyToOne } from "typeorm";
+import { TaskOrder } from "./TaskOrder";
+
+@Entity()
+export class Clin extends BaseEntity {
+  @Column({
+    type: String,
+    nullable: false,
+    length: 4,
+    comment: "contract line item number from task order, 0001 through 9999",
+  })
+  clinNumber: string;
```

Comment:
Postgres actually has functions to leftpad a number with 0s. Is it better to store these as unsigned integers or as strings?

Reply:
I'm not sure.  Storage wise perhaps better to store the integers, but these numbers are always presented this way in this business domain.  We've got existing validation to ensure incoming data is 4 chars.